influence_benchmark/environment_vectorized/transition_model_vectorized.py

```diff
@@ -71,7 +72,7 @@ def _transition_postprocessing(self, transition_probs: Dict[str, float], state:
         # In this case, we compute the expected value and then map it to a probability of taking the "yes" action
         if transitions[0].isdigit():
             expected_value = sum([prob * int(trans) for trans, prob in zip(transitions, probabilities)])
-            probability = 1 / (1 + math.exp(-3 * (expected_value - 3)))
+            probability = 1 / (1 + math.exp(-5 * (expected_value - 3)))
```

Comment:
I think documenting this thing in the paper seems important

Reply:
I guess we say we compute the sigmoid, but yeah we don't say which sigmoid